id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/batch/config/BatchConfiguration.java

```
@@ -83,29 +83,29 @@
 	 * Process job.
 	 */
 	@Scheduled(fixedDelayString = "${mosip.credential.request.job.timedelay}")
-	public void processJob() {
+	public void processJob() throws Exception {
 		try {
 			JobParameters jobParameters = new JobParametersBuilder().addLong("time", System.currentTimeMillis())
 					.toJobParameters();
 			jobLauncher.run(credentialProcessJob, jobParameters);
 
 		} catch (Exception e) {
-			e.printStackTrace();
+			throw new Exception();

```

Why do we need to rethrow the exception? What was the actual issue?